types/cordova-plugin-safariviewcontroller/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "max-line-length": false
```

Comment:
why do we need to turn off this rule?

Reply:
This is due to the very long links that I've included in the docs. They are part of block comments. I don't know of another way to disable linting rules without using `tslint:disable:[rule]` syntax which by itself is again forbidden by the linter. `tslint:disable-next-line:[rule]` won't cut it for a block comment.